### ISSUE TITLE
Calling `PandasBaseRenderer().render(data)` throws `TypeError`

### ISSUE THREAD

**Issue by arthurio** [wq/django-rest-pandas#34]
For personal reasons, I wanted to call the `PandasCSVRenderer` on a dataframe outside of a view.

Because `renderer_context` is defaulted to `None`, the following code will throw `TypeError: argument of type 'NoneType' is not iterable`:

```python
from rest_pandas import renderers


data_frame = pandas.DataFrame(some_data)
print renderers.PandasCSVRenderer().render(data_frame)
>>>TypeError: argument of type 'NoneType' is not iterable
```

I suggest two options to fix the following code:
https://github.com/wq/django-rest-pandas/blob/edd3225acb21fcb0e36d8889d7bdfa5dedc5d67b/rest_pandas/renderers.py#L30

1. Change the if statement:
```python
if renderer_context and 'response' in renderer_context:
```
2. Add the following at the top of the function:
```python
if renderer_context is None:
    renreder_context = {}
...
```

**Comment by sheppard** [comment]
Thanks for the suggestion.  This is fixed in [v1.1.0](https://github.com/wq/django-rest-pandas/releases/tag/v1.1.0)

**Comment by arthurio** [comment]
Thanks for closing the issue!
